### PR TITLE
scss styling

### PR DESCRIPTION
**Pull request by JacqAttaq**
I would keep working on it, but colors are meaningless right now.

### CODE REVIEW

File: app/assets/stylesheets/styles.scss [hunk 1]
```diff
@@ -40,25 +46,4 @@ li {
 .side-panel-image {
   text-align: center;
   margin: 1.5em;
-}
-
-.favorite-button, .unfavorite-button {
```
Comment (posgarou) [inline app/assets/stylesheets/styles.scss:45]: These styles are used on the School#show page for the heart icon.  Would you add them back in or modify them (as you see fit)?

Comment (JacqAttaq) [inline app/assets/stylesheets/styles.scss:45]: I don’t even remember touching that, sorry. I will add back in as soon as I can.

Comment (posgarou) [inline app/assets/stylesheets/styles.scss:45]: No worries!

Comment (JacqAttaq) [inline app/assets/stylesheets/styles.scss:45]: And all of the below needs to go back in? Does it matter what lines I put it on?

Comment (posgarou) [inline app/assets/stylesheets/styles.scss:45]: Yeah, all of those styles, sorry.  And no, doesn't matter in the slightest.

**Pull request merged**